src/sidebar/services/annotations-exporter.ts

```diff
@@ -39,4 +46,50 @@ export class AnnotationsExporter {
       ) as APIAnnotationData[],
     };
   }
+
+  buildTextExportContent(
+    annotations: APIAnnotationData[],
+    groupName = '',
+    /* istanbul ignore next - test seam */
+    now = new Date(),
+  ): string {
+    const [firstAnnotation] = annotations;
+    if (!firstAnnotation) {
+      throw new Error('No annotations to export');
+    }
+
+    const { uri, title } = documentMetadata(firstAnnotation);
+    const uniqueUsers = [
+      ...new Set(
+        annotations
+          .map(annotation => username(annotation.user))
+          .filter(Boolean),
+      ),
+    ];
+
+    const annotationsText = annotations
+      .map(
+        (annotation, index) =>
+          trimAndDedent`
+          Annotation ${index + 1}:
+          ${annotation.created}
```

Comment:
The date should be formatted in a locale-appropriate way, rather than the default ISO representation. In tests you can mock the APIs that do date formatting to avoid tests failing if run in different locales.